src/components/sidebar.tsx

```diff
@@ -25,10 +28,9 @@ export function Sidebar() {
   return (
     <div className="w-48 bg-gradient-to-b from-sideBar-Top from-0% to-sideBar-Bottom to-35% shadow-3xl flex flex-col justify-between text-center">
       <div className="flex flex-col items-center mt-6 text-white gap-1">
-        <img
-          src="https://avatars.githubusercontent.com/u/102186472?v=4"
-          alt="userPhoto"
-          className="rounded-full h-24 w-24 border-white border-2"
+        <Avatar
+          imageUrl={imageUrl === undefined ? user : imageUrl}
```

Comment:
Why are you doing it in here, if inside the Avatar component it's doing the same thing? If you configured the Avatar property imageUrl to be string | undefined, it is expected that the Avatar component will know how to deal with that, so in this case, you don't need to set a fallback at the parent component, just pass the imageUrl directly. Also, don't forget to remove the "user" image import, as it won't be used here anymore.

Reply:
thanks for that, i managed to get it outside